bundles/mapping/mapmodule/AbstractMapModule.js

```diff
@@ -940,7 +940,7 @@ Oskari.clazz.define(
 
           this.loadtimer = setTimeout( function() {
             var eventBuilder = Oskari.eventBuilder( 'ProgressEvent' );
-            var event = eventBuilder( done, 'maplayer' );
+            var event = eventBuilder(done, layerId);
```

Comment:
'maplayer' is used to detect _what_ is being loaded. Need to take a better look at it if we'll just change it like this or add layerId as third param. Currently it's under mapmodule which is probably a wrong place if it could be used more commonly.